### PR TITLE
add CSS Theme Selection environment variable

### LINKED ISSUE #743: An environment variable to configure the Setup Theme
### Feature description

An environment variable to configure the Setup Theme

### Motivation

v11 added css themes for setup, but there's not a way to configure them other than by enabling `CONTAINER_PRESERVE_CONFIG`.

### Example

There are 3 setup themes and the value is stored in the `cssTheme` key in `options.json`
- Foundry Virtual Tabletop: `foundry`
- Classic Fantasy: `fantasy`
- Science Fiction: `scifi`

### Pitch

It would be in line, design-wise with the other env vars.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by orangepeelbeef**
#  #

## 🗣 Description ##

Set an environment variable to select Setup page CSS theme

## 💭 Motivation and context ##

resolves #743 

## 🧪 Testing ##

Built the image from the develop branch, set the docker env variable and watched the CSS change

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [ ] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [ ] Tests have been added and/or modified to cover the changes in this PR.
- [ ] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Add a tag or create a release.

### CODE REVIEW

**Comment by orangepeelbeef** [comment]
I see that my IDE changed the format on the README.md that wasn't really intentional.  I can resubmit it without the format change, or i can let the IDE adjust the other tables as well.  Having them in different formats is probably not great.

it wants the table to look like
this
image
rather than
image



I don't think any of this is why the validation jobs are failing but let me know if you want it adjusted

File: README.md
Comment (felddy) [inline README.md:1]: Yes, Please limit your changes to the new line you are adding to the table.

File: src/set_options.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ let options: object = {
   awsConfig: process.env.FOUNDRY_AWS_CONFIG || null,
   compressSocket: process.env.FOUNDRY_COMPRESS_WEBSOCKET == "true",
   compressStatic: process.env.FOUNDRY_MINIFY_STATIC_FILES == "true",
+  cssTheme: process.env.FOUNDRY_CSS_THEME || "foundry",
```
Comment (felddy) [inline src/set_options.ts:44]: Looks good.  I thought the default should have been `null` but it is in fact `foundry`.  💪

**Review by felddy — CHANGES_REQUESTED** [review]

**Comment by felddy** [comment]
Sorry I've been neglecting this PR.   There was a (and still is) a big weird architecture issue that was blocking all the workflows up.   This is looking good.  See my one request to limit the README changes.

File: README.md [hunk 2]
```diff
@@ -276,6 +276,7 @@ secrets](#using-secrets) instead of environment variables.
 | `CONTAINER_VERBOSE` | Set to `true` to enable verbose logging for the container utility scripts. | `false` |
 | `FOUNDRY_ADMIN_KEY` | Admin password to be applied at startup.  If omitted the admin password will be cleared.  May be set [using secrets](#using-secrets). | |
 | `FOUNDRY_AWS_CONFIG` | An absolute or relative path that points to the [awsConfig.json](https://foundryvtt.com/article/aws-s3/) or `true` for AWS environment variable [credentials evaluation](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-credentials-node.html) usage. | `null` |
+| `FOUNDRY_CSS_THEME` | Choose the CSS Theme for the setup page in foundry.  Choose from foundry, fantasy, or scifi. | `foundry` |
```
Comment (felddy) [inline README.md]: Just a couple of formatting suggestions:

```suggestion
| `FOUNDRY_CSS_THEME` | Choose the CSS theme for the setup page.  Choose from `foundry`, `fantasy`, or `scifi`. | `foundry` |
```

**Review by felddy — CHANGES_REQUESTED** [review]
Thanks for the quick update.  Just one more suggestion.

**Review by felddy — APPROVED** [review]
Great.  Thanks.

**Pull request merged**